src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -41,8 +41,12 @@ internal class InProcLanguageServer
 
         private VSClientCapabilities _clientCapabilities;
 
-        public InProcLanguageServer(Stream inputStream, Stream outputStream, LanguageServerProtocol protocol,
-            CodeAnalysis.Workspace workspace, IDiagnosticService diagnosticService, string? clientName)
+        public InProcLanguageServer(Stream inputStream,
+            Stream outputStream,
+            LanguageServerProtocol protocol,
+            CodeAnalysis.Workspace workspace,
+            IDiagnosticService diagnosticService,
+            string? clientName)
         {
             _protocol = protocol;
             _workspace = workspace;
```

Comment:
for diagnostics and avoiding returning non-razor documents for razor requests.  hopefully can merge the razor one into the liveshare one and this can all go away